### ISSUE TITLE
Windows Server Insider build 16287 pull failure

### ISSUE THREAD

**Issue by kannus** [docker/for-win#1118]
### Expected behavior

After following the instructions to install Docker EE on Windows Server Insider build 16287, and the Linux Container Preview on it, the new dockerd started up fine.  The docker pull command should allow me to pull the busybox image from Docker hub.

docker.exe -H "npipe:////./pipe/docker_lcow" pull busybox

### Actual behavior

After following the instructions to install Docker EE on Windows Server Insider build 16287, and the Linux Container Preview on it, the new dockerd started up fine.  However, when I tried to pull the busybox image from Docker hub, it fails to register the image.

### Information

  - Diagnostic ID from "Diagnose & Feedback" in the menu.
No UI, not available
  - a reproducible case if this is a bug, Dockerfiles FTW
  - page URL if this is a docs issue or the name of a man page
  - host distribution and version (Windows version, build number, etc)

### Steps to reproduce the behavior

  1. Install Windows Server Insider build 16287
  2. Install Docker EE on this server
  3. Follow the procedure from [Preview Page](https://blog.docker.com/2017/09/preview-linux-containers-on-windows/)
   4. It cannot install/register the busybox image

**Comment by kannus** [comment]
Adding the logs from shells.
[Docker Error.txt](https://github.com/docker/for-win/files/1325546/Docker.Error.txt)

**Comment by friism** [comment]
You need to set the environment variable and run the daemon with the `--experimental` flag.

    $Env:LCOW_SUPPORTED=1
    .\dockerd.exe --experimental

**Comment by kannus** [comment]
Yes I did that from the profile.ps1 for client. For dockerd, it was launched through another ps1 script with the environment and experimental command line switch there.

**Comment by friism** [comment]
Can you try launching it directly as described in the blog post and see if that works?

**Comment by kannus** [comment]
Tried it and made sure $Env:LCOW_SUPPORTED is 1 but got the same error.  One thing I probably should've mentioned is, I ran the Insider Preview Server Core as a VM within Hyper-V.  The nested virtualization is turned on for this VM.  Could this be the culprit?  Also, is there a minimum memory requirement?  I used to assign 4GB for this VM but "docker pull" resulted in "not enough storage" failure so I bumped it up to 6GB and got the reported error instead.
